e2e/cephfs_helper.go

```diff
@@ -147,3 +150,43 @@ func getSubvolumePath(f *framework.Framework, filesystem, subvolgrp, subvolume s
 	}
 	return strings.TrimSpace(stdOut), nil
 }
+
+func getSnapName(snapNamespace, snapName string) (string, error) {
+	sclient, err := newSnapshotClient()
+	if err != nil {
+		return "", err
+	}
+	snap, err := sclient.SnapshotV1beta1().VolumeSnapshots(snapNamespace).Get(context.TODO(), snapName, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	sc, err := sclient.SnapshotV1beta1().VolumeSnapshotContents().Get(context.TODO(), *snap.Status.BoundVolumeSnapshotContentName, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	snapIDRegex := regexp.MustCompile(`(\w+\-?){5}$`)
+	snapID := snapIDRegex.FindString(*sc.Status.SnapshotHandle)
+	snapshotName := fmt.Sprintf("csi-snap-%s", snapID)
+	e2elog.Logf("snapshotName= %s", snapshotName)
+	return snapshotName, nil
+}
+
+func deleteBackingCephFSSubvolumeSnapshot(f *framework.Framework, pvc *v1.PersistentVolumeClaim, snap *snapapi.VolumeSnapshot) error {
+	snapshotName, err := getSnapName(snap.Namespace, snap.Name)
+	if err != nil {
+		return err
+	}
+	imageData, err := getImageInfoFromPVC(pvc.Namespace, pvc.Name, f)
+	if err != nil {
+		return err
+	}
+	cmd := fmt.Sprintf("ceph fs subvolume snapshot rm myfs %s %s %s", imageData.imageName, snapshotName, subvolumegroup)
```

Comment:
Instead of hard-coding `myfs`, you can use the global variable `fileSystemName`.